### PR TITLE
BOOKKEEPER-907: EntryFormatter should be configur.

### PR DESCRIPTION
**Pull request by reddycharan**
for ReadLedgerEntriesCmd, EntryFormatter should
be configurable and HexDumpEntryFormatter should
be one of them.

### CODE REVIEW

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/util/HexDumpEntryFormatter.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package org.apache.bookkeeper.util;
```
Comment (sijie) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/util/HexDumpEntryFormatter.java]: missing apache header

Comment (reddycharan) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/util/HexDumpEntryFormatter.java]: added header

**Comment by zzsongsg** [comment]
+1 LGTM

**Comment by sijie** [comment]
+1 merged in master as ca7e5fd

**Pull request closed without merging**